pxf/resources/META-INF/LICENSE

```diff
@@ -0,0 +1,380 @@
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=======================================================================
+
+Apache HAWQ (incubating) - contained works supporting PXF components:
+
+This product contains additional works that are distributed under
+Apache License, Version 2.0 (ALv2).
+
+    Apache Tomcat 7.0.62
+
+=======================================================================
+
+Apache HAWQ (incubating) PXF runtime dependent subcomponents:
+
+  The Apache HAWQ (incubating) project contains PXF. It is an
+  extensible framework that allows HAWQ to query external data files,
+  whose metadata is not managed by HAWQ. PXF includes built-in
+  connectors for accessing data that exists inside HDFS files, Hive
+  tables, HBase tables and more. Users can also create their own
+  connectors to other data storages or processing engines. To create
+  these connectors using JAVA plugins, see the PXF API and Reference
+  Guide online.
+
+  The convenience binaries require the following hadoop components are
+  higher:
+
+    hadoop: 2.7.1
+    hadoop-hdfs: 2.7.1
+    hadoop-mapreduce: 2.7.1
+
+  The runtime dependencies (below) are accessed via classpath
+  configurations from the hadoop component distributions. They are not
+  distributed with these convenience binary packages.
+
+    ST4, 4.0.4
```

Comment:
The runtime which are not distributed with the package should not be listed here?